validator/README.md

```diff
@@ -19,30 +19,61 @@ limitations under the License.
 A validator for the
 [AMP HTML format](https://github.com/ampproject/amphtml/blob/master/README.md).
 
+To use, install Node.js version 4.X on your system. E.g.,
+[by downloading](https://nodejs.org/en/download/) or
+[by using a package manager](https://nodejs.org/en/download/package-manager/) or
+[by using NVM](https://github.com/creationix/nvm).
+
 ## Using the command-line tool
 
-* Install Node.js version 4.X on your system. E.g.,
-  [by downloading](https://nodejs.org/en/download/) or
-  [by using a package manager](https://nodejs.org/en/download/package-manager/) or
-  [by using NVM](https://github.com/creationix/nvm).
-* Type `node index.js` in this directory to get started:
+* Type `./index.js` in this directory to get started:
 
 ```
-$ node index.js testdata/feature_tests/minimum_valid_amp.html
+$ ./index.js testdata/feature_tests/minimum_valid_amp.html
 testdata/feature_tests/minimum_valid_amp.html: PASS
 
-$ node index.js testdata/feature_tests/several_errors.html
+$ ./index.js testdata/feature_tests/several_errors.html
 testdata/feature_tests/several_errors.html:23:2 The attribute 'charset' may not appear in tag 'meta name= and content='.
 testdata/feature_tests/several_errors.html:26:2 The tag 'script' is disallowed except in specific forms.
 testdata/feature_tests/several_errors.html:32:2 The mandatory attribute 'height' is missing in tag 'amp-img'. (see https://www.ampproject.org/docs/reference/amp-img.html)
 testdata/feature_tests/several_errors.html:34:2 The attribute 'width' in tag 'amp-ad' is set to the invalid value '100%'. (see https://www.ampproject.org/docs/reference/amp-ad.html)
 ...
 ```
 
-* If you wish to install the Validator as a system command,
-  install the NPM package manager (e.g. using apt-get in Ubuntu Linux) and
-  run `npm install -g` in this directory. After that, you may type
-  `amp_validator` in any directory to invoke the validator.
+If you wish to install the Validator as a system command,
+install the NPM package manager (e.g. using apt-get in Ubuntu Linux) and
+run `npm install -g` in this directory. After that, you may type
+`amp_validator` in any directory to invoke the validator.
+
+## Using the web UI
+
+This is a simple web editor which validates AMP HTML documents on the fly.
+This feature is new and experimental, feedback is especially welcome.
+
+* Type `./index.js webui` in this directory.
+* Point your web browser to http://127.0.0.1:8765/
+
+## Using the NodeJS API
+
+This API is new and still experimental, feedback is especially welcome.
+
+```
+'use strict';
+
+const ampValidator = require('amp_validator');
```

Comment:
amp-validator is taken (https://www.npmjs.com/search?q=amp-validator) but I didn't pick the underscore because of that. '-' isn't allowed in google3 dir names (for good reasons btw. - in Python, package names are identifiers ...). So, amp_validator looked simple for consistency.

I think can think of these choices:
- Leave amp_validator as is (and consider it tbd)
- Use amphtml-validator for now
- Use amp-validator despite the existing package

@cramforce might have an opinion on this naming issue as well.